packages/scandipwa/src/component/Header/Header.container.js

```diff
@@ -412,13 +426,18 @@ export class HeaderContainer extends NavigationAbstractContainer {
     }
 
     onSignIn() {
+        const { navigationState: { title }, totals: { is_virtual } } = this.props;
         const { location: { pathname } } = history;
 
         goToPreviousNavigationState();
 
         if (pathname.includes(CHECKOUT_URL)) {
             this.setState({ showMyAccountLogin: false });
         }
+
+        if (pathname.includes(BILLING_URL) && title === 'Sign in' && !is_virtual) {
```

Comment:
Definitely should not really on title check, it can be translated and on other langue it will not be 'Sign in'.